Scene/scene.js

```diff
@@ -74,6 +74,13 @@ class Scene {
         return id;
     }
 
+    async UpdateResolution(resolution) {
+        // get the current scene id
```

Comment:
The scene itself doesn't have an ID. The models in the scene have IDs.
The IDs are just numbers that index into the `Scene._models` object

quick example: when id = 1, then you access model 1 by doing `Scene._models[1]`